classes/Domain/Model/TalkComment.php

```diff
@@ -1,6 +1,5 @@
 <?php
 
-
```

Comment:
Looks like we definitely have a good case for enabling the `single_blank_line_before_namespace` fixer!

👍